test/pytest/test_pytorch_profiler.py

```diff
@@ -137,17 +138,22 @@ def test_batch_input(set_custom_handler, handler_name):
     """
     Tests pytorch profiler integration with batch inference
     """
+
     CUSTOM_PATH = "/tmp/output/resnet-152-batch"
+
     if os.path.exists(CUSTOM_PATH):
         shutil.rmtree(CUSTOM_PATH)
     assert os.path.exists(data_file_resnet)
 
-    cmd = ["bash", os.path.join(profiler_utils, "resnet_batch.sh")]
+    def invoke_batch_input():
+        data = open(data_file_resnet, "rb")
+        response = requests.post("{}/predictions/resnet152".format(TF_INFERENCE_API), data)
+        print(response.text)
```

Comment:
Could you add response validation instead of printing?

Reply:
Done.